public/index.html

```diff
@@ -4,7 +4,10 @@
 
     <style>
       @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,100&family=Inter&display=swap');
-      </style>
+    </style>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Fjalla+One&family=Merriweather+Sans&display=swap');
+    </style>
```

Comment:
Comment: I think @NurkaAmre has a PR in for updating the fonts. Let's make sure you don't step on each other's work.